go/libraries/doltcore/sqle/read_replica_database.go

```diff
@@ -302,6 +302,8 @@ func pullBranchesAndUpdateWorkingSet(
 	}
 
 	// update the current working set if necessary
+	// TODO: the current ref is wrong?
+	// Or we need to always update working sets?
```

Comment:
Is this TODO still needed?

Reply:
Nope, that whole code block is obsolete, good catch!